tools/rostest/nodes/paramtest

```diff
@@ -90,7 +90,7 @@ class ParamTest(unittest.TestCase):
 
         wallclock_timeout_t = time.time() + wait_time
         param_obtained = None
-        while not param_obtained and time.time() < wallclock_timeout_t:
+        while param_obtained in None and time.time() < wallclock_timeout_t:
```

Comment:
Please use `is None` here. Currently the test is failing because `None` is not iterable. Please make sure to run the code with your changes locally to confirm it works as desired.